config/socket_windows.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+func setTransportAddress(cfg *Config) {
+	cfg.TransportAddress = `\\.\pipe\manifoldco\torus`
```

Comment:
From what I get from [the docs](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365783(v=vs.85).aspx) this is system wide. Thinking back now, this probably goes against what we're trying to achieve with Unix Sockets.

Looking back at how Keybase do it, they make a string (`manifoldco\torus` in our case) based on the homedir etc. This makes it unique but still "guessable". I'll retest this again because re-reading the docs, it shouldn't work with backslashes.